apps/src/applab/designElements/ColorPickerPropertyRow.jsx

```diff
@@ -28,7 +28,7 @@ var ColorPickerPropertyRow = React.createClass({
    * Make our button a colpick color picker, if it isn't already
    */
   ensureColorPicker: function () {
-    var element = React.findDOMNode(this.refs.colorPicker);
+    var element = ReactDOM.findDOMNode(this.refs.colorPicker);
```

Comment:
Note: I think we can get away with removing ReactDOM.findDOMNode completely in instances like this now, i.e. `this.refs.colorPicker` already gives us a DOM node in v0.14. Not sure if it's worth changing all of these instances.